unison-cli/src/Unison/Share/Sync.hs

```diff
@@ -74,13 +75,13 @@ import Unison.Util.Monoid (foldMapM)
 
 -- | The maximum number of downloader threads, during a pull.
 maxSimultaneousPullDownloaders :: Int
-maxSimultaneousPullDownloaders = 5
+maxSimultaneousPullDownloaders = 1
 
 -- | The maximum number of push workers at a time. Each push worker reads from the database and uploads entities.
 -- Share currently parallelizes on it's own in the backend, and any more than one push worker
 -- just results in serialization conflicts which slow things down.
 maxSimultaneousPushWorkers :: Int
-maxSimultaneousPushWorkers = 5
+maxSimultaneousPushWorkers = 1
```

Comment:
Did we wanna leave these at 1?

Reply:
Oh whoops, we want to leave `pull` at 5, but push goes to 1 👍🏼